### PR TITLE
drop create-react-app + swc in-repository test

### PR DESCRIPTION
**Pull request by n1ru4l**
this replaces creatr-react-app examples with vite
It also adds an in-repository example for verifying that the swc plugin actually works.

### CODE REVIEW

**Comment by JesseVelden** [comment]
The strange thing is that I cannot reproduce the issue locally. I will do some more digging

**Comment by n1ru4l** [comment]
@JesseVelden @YassinEldeeb pinned this down to the rust version

**Comment by JesseVelden** [comment]
What I also saw is that we need access to the namingConvention, as now the  `const allFilmsWithVariablesQueryDocument =` query example doesn't resolve to `AllFilmsWithVariablesQueryDocument` (note the uppercase A). 

I was assuming that you just need to do that. I guess we can provide it as an option to the SWC plugin, or warn the user instead. I'm not sure how the Babel plugin does that.

**Comment by JesseVelden** [comment]
> @JesseVelden @YassinEldeeb pinned this down to the rust version

Oh lol, I wouldn't expect that. Great debugging work @YassinEldeeb !

File: packages/presets/swc-plugin/tests/fixtures/simple-uppercase-operation-name.other-dir.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */ //@ts-ignore
-import { CFragmentDoc } from "../graphql";
-import { BDocument } from "../graphql";
-import { ADocument } from "../graphql";
+import { CFragmentDoc } from "./../graphql";
+import { BDocument } from "./../graphql";
+import { ADocument } from "./../graphql";
```
Comment (n1ru4l) [inline packages/presets/swc-plugin/tests/fixtures/simple-uppercase-operation-name.other-dir.js:4]: @YassinEldeeb The `./` is kind of redundant as `./../graphql` is the same as `../graphql` 🤔

Comment (n1ru4l) [reply packages/presets/swc-plugin/tests/fixtures/simple-uppercase-operation-name.other-dir.js:4]: we can improve it in a follow-up PR

**Comment by JesseVelden** [comment]
Great work guys! I hadn't much time to work on it this week

**Pull request merged**